lib/recording/RecordingManager.js

```diff
@@ -291,53 +304,59 @@ class RecordingManager {
             logger.error("Failed to changeRecordedEntry: %s", ErrorUtils.error2string(err))
         }
     }
-    stopRecording(entryId, recordedEntryId, recordingSessionPath){
 
-        let doneFilePath = path.join(recordingSessionPath, 'done')
-        let recordingEntrySession = this.recordingList[entryId];
-        if (!recordingEntrySession) {
-            logger.warn("[%s][stopRecording]  Can't stop recording, entry doesn't exist on recording list", entryId);
-            return
-        }
-        return recordingEntrySession.serializedActionsPromise.finally(() =>{ // check first that all chunks are completed
-            let totalAddedChunks = _.reduce(recordingEntrySession.flavorsChunksCounter,(memo,flavorChunks)=>{
-                return memo+ flavorChunks
-            }, 0)
-            //todo should also print warning if # of chunks is not equall between all flavors!
-            let recordingSessionDuration = recordingEntrySession.getDuration() - recordingEntrySession.startDuration
-            if (! recordingSessionDuration > 0 ){
-                logger.warn("[%s-%s][stopRecording] Entry duration hasn't changed, do nothing. start [%s] current[%s], totalAddedChunks [%s]", entryId, recordedEntryId, recordingEntrySession.getDuration(), recordingEntrySession.startDuration , totalAddedChunks)
-                delete this.recordingList[entryId];
-                return qio.write(doneFilePath, 'done')
+    stopRecording(entryId) {
```

Comment:
General comment:
stopRecording return promise that caller function doesn't handle. Please update code and test.